### PR TITLE
games-action/teeworlds: bump to 0.7.5

### PR DESCRIPTION
**Pull request by ktrace**
Bump version
Remove dev-util/bam (switch to cmake)

Closes: https://bugs.gentoo.org/668928
Signed-off-by: Victor Kustov <ktrace@yandex.ru>

### CODE REVIEW

File: games-action/teeworlds/teeworlds-0.7.3.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (SoapGentoo) [inline games-action/teeworlds/teeworlds-0.7.3.1.ebuild]: space after `EAPI`

File: games-action/teeworlds/teeworlds-0.7.3.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+PYTHON_COMPAT=( python2_7 )
+inherit eutils cmake-utils multiprocessing python-any-r1
```
Comment (SoapGentoo) [inline games-action/teeworlds/teeworlds-0.7.3.1.ebuild]: why are you inheriting `eutils` and `multiprocessing`?

File: games-action/teeworlds/teeworlds-0.7.3.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+RDEPEND="
+	!dedicated? (
+		app-arch/bzip2
+		media-libs/freetype
+		media-libs/libsdl[X,sound,opengl,video]
+		media-libs/pnglite
+		media-sound/wavpack
+		virtual/glu
+		virtual/opengl
+		x11-libs/libX11 )
```
Comment (SoapGentoo) [inline games-action/teeworlds/teeworlds-0.7.3.1.ebuild]: the indentation here is super confusing:
```
RDEPEND="
	!dedicated? (
		app-arch/bzip2
		media-libs/freetype
		media-libs/libsdl[X,sound,opengl,video]
		media-libs/pnglite
		media-sound/wavpack
		virtual/glu
		virtual/opengl
		x11-libs/libX11
	)
	sys-libs/zlib"
```

**Comment by thesamesam** [comment]
Could you bump this to 0.7.5?

**Pull request closed without merging**